CMakeLists.txt

```diff
@@ -408,10 +422,13 @@ SET(CPACK_PACKAGE_INSTALL_DIRECTORY "Hydrogen")
 IF(MINGW)
     #Set the other files that will be used in CPack
     SET(WIN64 "OFF" CACHE BOOL "Windows 64 Bit")
-	SET(CPACK_STRIP_FILES "${WINDOWS_DIR}/src/cli/h2cli.exe" "${WINDOWS_DIR}/src/gui/hydrogen.exe" "${WINDOWS_DIR}/src/player/h2player.exe")
+    if (CMAKE_BUILD_TYPE MATCHES Release)
+         SET(CPACK_STRIP_FILES TRUE)
+    else()
+         SET(CPACK_STRIP_FILES FALSE)
+    endif()
```

Comment:
On recent CMake (>2.6),  [`CPACK_STRIP_FILES`](https://cmake.org/cmake/help/v3.14/module/CPack.html#variable:CPACK_STRIP_FILES) is boolean.